internal/dcache/rpc/server/handler.go

```diff
@@ -1063,17 +999,16 @@ func (h *ChunkServiceHandler) StartSync(ctx context.Context, req *models.StartSy
 		return nil, rpc.NewResponseError(rpc.NeedToRefreshClusterMap, fmt.Sprintf("request component RVs are invalid for MV %s [%v]", req.MV, err.Error()))
 	}
 
-	// Set IO quiescing in the mv. Now GetChunk, PutChunk, will not allow any new IO.
+	// acquire write lock on the opMutex for this MV. Now GetChunk, PutChunk and RemoveChunk will not allow any new IO.
 	// Also wait for any ongoing IOs to complete.
-	err := mvInfo.quiesceIOsStart()
-	common.Assert(err == nil, fmt.Sprintf("failed to quiesce IOs for MV %s [%v]", req.MV, err))
+	mvInfo.acquireSyncOpWriteLock()
 
```

Comment:
you may want a debug log after the sync acquires the lock

Reply:
Added debug logs